.github/workflows/check-entities.yml

```diff
@@ -0,0 +1,51 @@
+name: Check DB Entities
+
+on: [pull_request]
+
+jobs:
+  check-db-entities:
+    name: Check DB Entities
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: Set up JDK 11
+        uses: actions/setup-java@v2
+        with:
+          distribution: adopt
+          java-version: 11
+      - name: Find touched DB Entities
+        id: touchedEntities
+        continue-on-error: true
+        run: |
+          status=$(grep '@Entity' -rinl . | xargs -r git diff-index --name-only --exit-code --output=touchedEntities origin/${{ github.base_ref }} $(); echo $?)
+          output=$(cat touchedEntities)
+          output="${output//'%'/'%25'}"
+          output="${output//$'\n'/'%0A'}"
+          output="${output//$'\r'/'%0D'}"
```

Comment:
Whenver we pass a multiline output from one step to another, it looses multiline structure, to fix it we need to replace some characters.
More info [here](https://github.com/orgs/community/discussions/26288)